src/components/layout/contextFilters/contextFilters.scss

```diff
@@ -18,7 +18,17 @@ $borderColor: #454a4e;
 
     .filter-icon { width: 13px; }
 
+    .filter-icon-disabled {
```

Comment:
The modularity here and in `select-container` is incorrect. You are duplicating code for both classes when you should be using compound classes.

```
.filter-icon { 
    width: 13px; 

    &.disabled { opacity: 0.5; }
}

.select-container { 
    width: 200px; 

    &.disabled { margin-top: -4px; }
}
```